### PR TITLE
(#5) Keyword Search

### PR DESCRIPTION
**Pull request by karlikpj**
Closes(#5) 

* Keyword Search Results

### CODE REVIEW

File: cypress/integration/common/index.js [hunk 1]
```diff
@@ -167,7 +173,8 @@ Then(
 );
 
 And('{string} radio is selected by default', (startWith) => {
-	cy.get(`input[value="Begins"]`).should('be.checked');
+	const searchMode = startWith !== 'Contains' ? 'Begins' : 'Contains';
+	cy.get(`input[value="${searchMode}"]`).should('be.checked');
```
Comment (kate-mashkina) [inline cypress/integration/common/index.js:177]: Can you revert this? I missed it while working on the tests...

File: cypress/integration/SearchResults.feature [hunk 2]
```diff
@@ -0,0 +1,83 @@
+Feature: As a user, I would like to be able to enter keywords and have the option to submit a search
+
+  # Scenario: User enters /search in the URL and tries to access the dictionary
+  #   Given the user navigates to bad search url "/search"
```
Comment (kate-mashkina) [inline cypress/integration/SearchResults.feature:4]: Were you able to make this test work?

**Review by kate-mashkina — APPROVED** [review]

**Review by seyilonge-nci — APPROVED** [review]

**Comment by lburack** [comment]
Also tagging @VictoriaSunNIH for review. I won't be able to get to this until end of day Friday.

**Comment by VictoriaSunNIH** [comment]
This looks good to me - marking as passed.

Tagging @lburack , @mworrest , @blilianyu

File: src/components/atomic/SynonymList/synonym-list.scss [hunk 3]
```diff
@@ -0,0 +1,39 @@
+ul.search-results-alias-list {
```
Comment (arcepaul) [inline src/components/atomic/SynonymList/synonym-list.scss:1]: can we sass-ify this

File: src/components/molecules/SearchTermListItem/search-term-list-item.jsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SynonymList from '../../atomic/SynonymList';
+
+import { Link } from 'react-router-dom';
+
+import './search-term-list-item.scss';
```
Comment (arcepaul) [inline src/components/molecules/SearchTermListItem/search-term-list-item.jsx:8]: Reference to the scss partial should be in the dictionaries.scss.  Now I'm seeing that this hasn't been done for other views and components.  Can we make some adjustments here?

**Review by arcepaul — APPROVED** [review]

**Pull request merged**